src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+const populations = document.querySelectorAll('.population');
+let totalValue = 0;
+
+for (let i = 0; i < populations.length; i++) {
+  const value = populations[i].textContent.replace(/,/g, '');
+
+  if (!isNaN(Number(value))) {
+    totalValue += Number(value);
+  }
+
+  const averageValue = totalValue / populations.length;
+
+  document.querySelector('.total-population').textContent =
+    totalValue.toLocaleString('en-US');
+
+  document.querySelector('.average-population').textContent =
+    averageValue.toLocaleString('en-US');
```

Comment:
Updating the DOM with the total and average population values should be done after the loop. This will ensure that the correct total and average values are displayed only once, after all calculations are complete.